src/components/[guild]/EditGuild/hooks/useEditGuild.ts

```diff
@@ -250,31 +261,55 @@ const useEditGuild = ({ onSuccess, guildId }: Props = {}) => {
         })
       } else {
         if (admin.creations.failedCount > 0) {
-          showErrorToast("Failed to create some admins")
+          showErrorToast({
```

Comment:
I know it is not really the scope, but I think in the bigger if condition  above, we need a `(!guildUpdateResult || (!!guildUpdateResult && !guildUpdateResult.error))` at the end instead of just `guildUpdateResult`
Because if a guild update doesn't happen, and other requests succeed, then `guildUpdateResult` is undefined, and we won't show a toast

Reply:
Oh right, nice catch! I also noticed an issue in `useEditRole`, and fixed that in my latest commit too!